### PR TITLE
update cluster check setting using http settings endpoint

### PR DESCRIPTION
**Pull request by BelSasha**

### CODE REVIEW

**Comment by BelSasha** [comment]
* **#898** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/898?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @BelSasha and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: runhouse/servers/cluster_servlet.py [hunk 1]
```diff
@@ -280,6 +256,13 @@ async def aperiodic_status_check(self):
                 )
                 await asyncio.sleep(INCREASED_STATUS_CHECK_INTERVAL)
             else:
+                # make sure that the thread will go to sleep, even if the interval size == -1
```
Comment (rohinb2) [inline runhouse/servers/cluster_servlet.py:259]: this is because we're reusing this thread for autostop too, right?

**Review by rohinb2 — APPROVED** [review]
awesome job. this is exactly what we wanted

File: runhouse/servers/cluster_servlet.py (hunk 1 above)
Comment (BelSasha) [reply runhouse/servers/cluster_servlet.py:259]: yeah, exactly @rohinb2

**Pull request merged**